src/components/FollowButton/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from 'react';
+import { useRecoilValue } from 'recoil';
+
+import { useCreateFollow, useDeleteFollow } from '@/hooks/useFolllow';
+import { useModal } from '@/hooks/useModal';
+import { userAtom } from '@/recoil/user';
+import { ModalType } from '@/types/modal';
+
+import { Button } from './style';
+
+export default function FollowButton(props: any) {
```

Comment:
요부분 interface로 타입을 지정해주면 any를 제거할 수 있을 것 같아요!

```suggestion
interface FollowButtonProps {
  followers: Follow[];
  userId: string;
}

export default function FollowButton({ followers, userId }: FollowButtonProps) {
```
props를 구조분해할당을 해서
아래의 props.followers 부분도 followers로 변경해볼 수 있겠네요 ㅎㅎ

Reply:
구조 분해 할당으로 리팩토링 하는 것도 필요하겠네요 ㅎㅎ 한번 날잡고 바꿔보겠습니다~!!